<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>496</width>
    <height>672</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QToolButton" name="removeAnalysisConfigurationButton">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="Experiment.qrc">
         <normaloff>:/openicon/icons/edit-clear-locationbar-ltr.png</normaloff>:/openicon/icons/edit-clear-locationbar-ltr.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>16</width>
         <height>16</height>
        </size>
       </property>
       <property name="autoRaise">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="analysisConfigurationComboBox">
       <property name="contextMenuPolicy">
        <enum>Qt::ActionsContextMenu</enum>
       </property>
       <property name="editable">
        <bool>true</bool>
       </property>
       <property name="maxVisibleItems">
        <number>100</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="saveButton">
       <property name="toolTip">
        <string>Save current</string>
       </property>
       <property name="text">
        <string notr="true">...</string>
       </property>
       <property name="icon">
        <iconset resource="Experiment.qrc">
         <normaloff>:/kde/icons/filesave.png</normaloff>:/kde/icons/filesave.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>16</width>
         <height>16</height>
        </size>
       </property>
       <property name="autoRaise">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="reloadButton">
       <property name="toolTip">
        <string>Save current</string>
       </property>
       <property name="text">
        <string notr="true">...</string>
       </property>
       <property name="icon">
        <iconset resource="Experiment.qrc">
         <normaloff>:/openicon/icons/document-revert-5.png</normaloff>:/openicon/icons/document-revert-5.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>16</width>
         <height>16</height>
        </size>
       </property>
       <property name="autoRaise">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QToolButton" name="addAnalysisButton">
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="Experiment.qrc">
         <normaloff>:/openicon/icons/edit-add.png</normaloff>:/openicon/icons/edit-add.png</iconset>
       </property>
       <property name="autoRaise">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="removeAnalysisButton">
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="Experiment.qrc">
         <normaloff>:/openicon/icons/edit_remove.png</normaloff>:/openicon/icons/edit_remove.png</iconset>
       </property>
       <property name="autoRaise">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="fitAllButton">
       <property name="text">
        <string>Fit All</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <widget class="TableViewExtended" name="analysisTableView"/>
     <widget class="QWidget" name="layoutWidget">
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QLabel" name="currentEvaluationLabel">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_3">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QLabel" name="descriptionLabel">
         <property name="text">
          <string>Description</string>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
         <property name="textInteractionFlags">
          <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByMouse</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="TableViewExtended" name="parameterTableView"/>
       </item>
      </layout>
     </widget>
     <widget class="TableViewExtended" name="resultsTableView"/>
     <widget class="TableViewExtended" name="pushTableView"/>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QCheckBox" name="checkBoxUseSmartStartValues">
       <property name="text">
        <string>Use Smart Start Values</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="useErrorBarsCheckBox">
       <property name="text">
        <string>Use Errorbars</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_4">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>37</width>
         <height>21</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="getSmartStartButton">
       <property name="text">
        <string>Smart to Start</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="fitButton">
       <property name="text">
        <string>Fit</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="plotButton">
       <property name="text">
        <string>Plot</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="removePlotButton">
       <property name="minimumSize">
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
       <property name="toolTip">
        <string>remove fit or plot from trace</string>
       </property>
       <property name="statusTip">
        <string/>
       </property>
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="Experiment.qrc">
         <normaloff>:/openicon/icons/edit-delete-6.png</normaloff>:/openicon/icons/edit-delete-6.png</iconset>
       </property>
       <property name="autoRaise">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="extractButton">
       <property name="minimumSize">
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
       <property name="toolTip">
        <string>copy the fit from the first selected trace</string>
       </property>
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="Experiment.qrc">
         <normaloff>:/openicon/icons/color-picker-black.png</normaloff>:/openicon/icons/color-picker-black.png</iconset>
       </property>
       <property name="autoRaise">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="fitToStartButton">
       <property name="minimumSize">
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
       <property name="toolTip">
        <string>copy fitted values to start values</string>
       </property>
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="Experiment.qrc">
         <normaloff>:/openicon/icons/arrow-left-2.png</normaloff>:/openicon/icons/arrow-left-2.png</iconset>
       </property>
       <property name="autoRaise">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QToolButton" name="addPushButton">
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="Experiment.qrc">
         <normaloff>:/openicon/icons/edit-add.png</normaloff>:/openicon/icons/edit-add.png</iconset>
       </property>
       <property name="autoRaise">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="removePushButton">
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="Experiment.qrc">
         <normaloff>:/openicon/icons/edit_remove.png</normaloff>:/openicon/icons/edit_remove.png</iconset>
       </property>
       <property name="autoRaise">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="pushButton">
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="Experiment.qrc">
         <normaloff>:/kde/icons/continue.png</normaloff>:/kde/icons/continue.png</iconset>
       </property>
       <property name="autoRaise">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>TableViewExtended</class>
   <extends>QTableView</extends>
   <header>uiModules.TableViewExtended</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="Experiment.qrc"/>
 </resources>
 <connections/>
</ui>
